src/mail.sh

```diff
@@ -1056,7 +1057,14 @@ function parse_mail_options()
           commit_count="${BASH_REMATCH[0]}"
           options_values['COMMIT_RANGE']+="$commit_count "
         fi
-        options_values['PASS_OPTION_TO_SEND_EMAIL']="$(str_strip "$* ${options_values['PATCH_VERSION']}")"
+        for i in "$@"; do
+          if [[ "${i}" =~ " " ]]; then
+            pass_option_to_send_email+=" ${i@Q}"
+          else
+            pass_option_to_send_email+=" ${i}"
+          fi
+        done
```

Comment:
It should be possible to change all of this to a single use of the `Q` operator, like so:
```
pass_option_to_send_email="${*@Q}"
```

Reply:
This change made a lot of test fail. I could not make reasonable changes to the tests to make it work. 
I'll try to look it later, but could you look the failed tests to make sure that the changes is not creating more bugs?

Reply:
@kwy95